src/test_solver.jl

```diff
@@ -0,0 +1,63 @@
+import Base.Test: @test, @test_approx_eq_eps
+
+const testset = [
```

Comment:
This is totally just a simple prototype.  This would be a cool idea, on the other hand, do we actually need that many different ODEs?  Is there anything that makes each of them independent enough to justify the presence of all of them?  If a solver can solve a harmonic oscillator to a reasonable precision then it surely will be able to solve other non-stiff equations.  Although, keeping additional ODEs doesn't seem to have any disadvantages to me: they don't impact the performance of the test suite that much.